backend/core/domains/bookingflow/migrations/0013_remove_payment_config_consolidated_fields.py

```
@@ -0,0 +1,50 @@
+        ),
+        migrations.AlterField(
+            model_name='paymentinfostepconfiguration',
+            name='available_payment_methods',
+            field=models.JSONField(blank=True, default=list, help_text='Available payment methods (deprecated)'),
+        ),
+        migrations.AlterField(
+            model_name='paymentinfostepconfiguration',
+            name='payment_terms',
+            field=models.TextField(blank=True, help_text='Payment terms text to display to clients'),
+        ),
+    ]
```

Ruff (0.13.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)